config/dev.exs

```diff
@@ -77,6 +77,8 @@ config :ret, Ret.Repo,
   database: "ret_dev",
   hostname: if(env_db_host == "", do: "localhost", else: env_db_host),
   template: "template0",
+  # Downloads from Sketchfab to file cache hold connections open
+  ownership_timeout: 60_000,
```

Comment:
Hmm. Is there a better solution to this that you're punting on for now?

Reply:
Ideally we could bump this dynamically for expected long-running transactions -- this is a mechanism to avoid pool depletion for hung processes. I think we are going to be stuck with picking a global value though. We need to hold the connection because we are using the database as a locking mechanism